Packs/RiskIQDigitalFootprint/IncidentFields/incidentfield-RiskIQ_Asset_GCP_Firewall_Name.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "incident_riskiqassetgcpfirewallname",
+    "version": -1,
+    "modified": "2020-07-15T12:00:31.496328399+05:30",
+    "name": "RiskIQ Asset GCP Firewall Name",
+    "ownerOnly": false,
+    "placeholder": "GCP Firewall Name for which the RiskIQAsset of the IP Address type is to be whitelisted.",
+    "description": "GCP Firewall Name for which the RiskIQAsset of the IP Address type is to be whitelisted.",
+    "cliName": "riskiqassetgcpfirewallname",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "script": "",
+    "fieldCalcScript": "",
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "selectValues": null,
+    "validationRegex": "",
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": null,
```

Comment:
Yes, it is purposeful. We did not associate it with any incident type since this will be used for a playbook that is triggered for indicators that will be executed using a job that can have any type of incident.